.travis.yml

```diff
@@ -5,7 +5,7 @@ before_install:
   - gem update --system --no-doc
   - gem install bundler --no-doc
   - gem update bundler --no-doc
-  - smoke/provision.sh
+  - bin/provision.sh
```

Comment:
Do you know if `provision.sh` becomes available to the gem users, if it is located in `bin/`? If so, this should either be renamed (to something like `influxdb-ruby-provision-test-server.sh`), or better, be moved to `ci/`.

Since the provisioning script is only required for Travis CI, I don't want users accidentally executing it.

Reply:
I understand. I'll check on this, and move it if necessary.